### DISCUSSION TITLE
error: in triplet x64-mingw-dynamic: Unable to find a valid Visual Studio instance

### DISCUSSION THREAD

**Discussion by Ultimecia1463** [microsoft/vcpkg#41823] [Q&A]
PS C:\Users\HP> vcpkg install sfml:x64-mingw-dynamic
Computing installation plan...
The following packages will be built and installed:
  * brotli:x64-mingw-dynamic@1.1.0#1
  * bzip2[core,tool]:x64-mingw-dynamic@1.0.8#6
  * egl-registry:x64-mingw-dynamic@2024-01-25
  * freetype[brotli,bzip2,core,png,zlib]:x64-mingw-dynamic@2.13.3
  * libflac[core,stack-protector]:x64-mingw-dynamic@1.4.3#2
  * libogg:x64-mingw-dynamic@1.3.5#1
  * libpng:x64-mingw-dynamic@1.6.43#3
  * libvorbis:x64-mingw-dynamic@1.3.7#3
  * openal-soft:x64-mingw-dynamic@1.23.1#2
  * opengl:x64-mingw-dynamic@2022-12-04#3
  * opengl-registry:x64-mingw-dynamic@2024-02-10#1
    sfml:x64-mingw-dynamic@2.6.1
  * stb:x64-mingw-dynamic@2024-07-29#1
  * vcpkg-cmake:x64-windows@2024-04-23
  * vcpkg-cmake-config:x64-windows@2024-05-23
Additional packages (*) will be modified to complete this operation.
error: in triplet x64-windows: Unable to find a valid Visual Studio instance
Could not locate a complete Visual Studio instance

how do i use x64-mingw-dynamic triplet 
ive even added it as default triplet in path

**Comment by dg0yt** [comment]
If you do not have Visual Studio, you probably want to change the host triplet. The target triplet will follow.

**Reply by Ultimecia1463** [reply]
thanks
